.github/workflows/pythonpublish.yml

```diff
@@ -1,6 +1,9 @@
 name: Upload Python Package
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - master
```

Comment:
Maybe uploading on tag would prevent mistakes?

https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-branches-and-tags

Reply:
Done 497ee70